torch_geometric/data/graph_store.py

```diff
@@ -104,21 +116,14 @@ def _put_edge_index(self, edge_index: EdgeTensorType,
 
     def put_edge_index(self, edge_index: EdgeTensorType, *args,
                        **kwargs) -> bool:
-        r"""Synchronously adds an edge_index tensor to the graph store.
+        r"""Synchronously adds the :obj:`edge_index` tensor to the graph store.
 
         Args:
-            tensor(EdgeTensorType): an edge_index in a format specified in
-            attr.
-            **attr(EdgeAttr): the edge attributes.
+            tensor (EdgeTensorType): The :obj:`edge_index` in the format
+                as specified in :obj:`attr`.
+            attr (EdgeAttr): The edge attributes.
```

Comment:
Just a random comment, `edge_attr` is quite  confusing here because we also use it for edge features

Reply:
Yes :( Do you know of a better name?

Reply:
perhaps `EdgeProperties` ... bit its a little verbose.

Reply:
Sounds good to me. `EdgeProp` also works. I will likely move this to a follow-up PR though if that works for you.

Reply:
👍